webapp/components/user_settings/user_settings_theme.jsx

```diff
@@ -180,10 +180,11 @@ export default class ThemeSetting extends React.Component {
         }
 
         const displayCustom = this.state.type === 'custom';
+        const AllowCustomThemes = global.window.mm_config.AllowCustomThemes !== 'false';
```

Comment:
Normally I'd complain that this doesn't start with a lowercase but not worth blocking merging for

Reply:
Fixed